src/Compression/CompressionCodecZSTD.cpp

```diff
@@ -156,6 +158,98 @@ void registerCodecZSTD(CompressionCodecFactory & factory)
     });
 }
 
+#ifdef ENABLE_QATZSTD_COMPRESSION
```

Comment:
Please move this into a new file, so we can maintain the one-class-one-file rule.

Reply:
Done